pyphare/pyphare/simulator/simulator.py

```
@@ -137,7 +137,7 @@
             self._auto_dump()  # first dump might be before first advance
 
             return self
-        except:
+        except (Exception, ...):
```

Ruff

`except` handlers should only be exception classes or tuples of exception classes

(B030)